### PR TITLE
remove poetry and add pipenv

### LINKED ISSUE #1: Rework python configuration to use pipenv and not poetry
Update readme to describe the new getting started.

### PR DESCRIPTION
**Pull request by miniscruff**
closes #1

### CODE REVIEW

File: web/Pipfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
```
Comment (pahartman1) [inline web/Pipfile:4]: Okay so we make a new source for pypi and this is where we will look for those packages? How did you make this file? just text editing? I haven't seen this before is it for pipenv? it looks like a list of lists of sources, list of packages etc.

**Review by pahartman1 — COMMENTED** [review]
Just a question on pipfile and also wondering what the lockfiles are and how they are created. looks good though.

File: web/Pipfile (hunk 1 above)
Comment (miniscruff) [reply web/Pipfile:4]: > Okay so we make a new source for pypi and this is where we will look for those packages?

yes, this allows for private registries as well as the main public one of PyPI.

> How did you make this file? 

It is generated when running `pipenv install <package>` for the first time.

> I haven't seen this before is it for pipenv?

Yes the Pipfile is the pipenv equivalent of a Package.json

>  it looks like a list of lists of sources, list of packages etc.

The [[source]] is the sources, later on is the packages with fast api and uvicorn

**Comment by miniscruff** [comment]
> Just a question on pipfile and also wondering what the lockfiles are and how they are created. looks good though.

Pipfile is basically a package.json, these are generated by pipenv for most of the commands.
Lock files are used to specify exactly what version, including hashes usually, of the packages and sub-packages of all dependencies are. This is to create reproducible builds regardless of external changes. This allows us to specify just the top level dependencies without having to include all sub-dependencies, etc.

**Comment by pahartman1** [comment]
thanks all makes sense now

**Review by pahartman1 — APPROVED** [review]

**Pull request merged**